packages/docusaurus-theme-classic/src/theme/Heading/index.js

```diff
@@ -18,7 +18,7 @@ const Heading = Tag => ({id, ...props}) => {
   return (
     <Tag {...props}>
       <a aria-hidden="true" tabIndex="-1" className="anchor" id={id} />
-      <a aria-hidden="true" tabIndex="-1" className="hash-link" href={`#${id}`}>
+      <a aria-hidden="true" tabIndex="-1" className="hash-link" href={`#${id}`} title="Direct link to section">
```

Comment:
I think `s/section/heading` might be more accurate. It'd be nice if we could make each title specific to the heading contents but that will be complicated and borderline unnecessary.

Reply:
Done :heavy_check_mark: